docs/intro.txt

```diff
@@ -2,11 +2,11 @@ pytket-cutensornet
 ==================
 
 ``pytket-cutensornet`` is an extension to ``pytket`` that allows ``pytket`` circuits and
-expectation values to be simulated using `cuTensorNet <https://docs.nvidia.com/cuda/cuquantum/cutensornet/index.html>`_.
+expectation values to be simulated using `cuTensorNet <https://docs.nvidia.com/cuda/cuquantum/latest/python/api/index.html#cutensornet-cuquantum-cutensornet>`_.
 
-`cuTensorNet <https://docs.nvidia.com/cuda/cuquantum/cutensornet/index.html>`_ is a
+`cuTensorNet <https://docs.nvidia.com/cuda/cuquantum/latest/python/api/index.html#cutensornet-cuquantum-cutensornet>`_ is a
```

Comment:
Should we leave the old link in this instance?

Reply:
Agreed, done